avocado/utils/ssh.py

```diff
@@ -214,3 +214,28 @@ def quit(self):
         :rtype: bool
         """
         return self._master_command('exit')
+
+    def copy_files(self, source, destination, recursive=False):
+        """
+        Copy Files to and from remote through scp session
```

Comment:
Missing a dot at the end of the line and missing a blank line after the first line.

https://www.python.org/dev/peps/pep-0257/